src/main/scala/org/ergoplatform/nodeView/wallet/IdUtils.scala

```diff
@@ -0,0 +1,18 @@
+package org.ergoplatform.nodeView.wallet
+
+import org.ergoplatform.settings.Algos
+
+object IdUtils {
```

Comment:
We already have these functions in `Scorex`.  May be the right place for this class is `scorex-utils` package.
Also we use supertagged to avoid type errors, in this case `EncodedBoxId` should not be used as `EncodedTokenId`